File: Backend-Java/src/main/java/org/example/DAL/entities/CommentImageEntity.java
package org.example.DAL.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@Table(name="tbl_comment_images")
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CommentImageEntity extends BaseEntity{
    @ManyToOne
    @JoinColumn(name="comment_id", nullable = false)
    private CommentEntity comment;
}

File: Backend-C#/Data.MyShop/Entities/CommentImageEntity.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data.MyShop.Entities
{
    [Table("tblCommentImages")]
    public class CommentImageEntity : BaseEntity<int>
    {
        //Foreign keys:

        //Every User have a company
        public CommentEntity Comment { get; set; }

        [ForeignKey(nameof(Comment))]
        public int? CommentId { get; set; }
    }
}
